qa/mc_sc_forging4.py

```diff
@@ -69,23 +69,23 @@ def run_test(self):
         # Generate 5 SC blocks
         scblock_ids = generate_next_blocks(sc_node, "first node", 5)
 
-        # Verify that SC block contains newly created MC blocks as MainchainHeaders and MainchainReferences
-        # First 4 SC blocks. Every block contain 3 MainchainReferences and 49 MainchainHeaders
+        # Verify that SC block contains newly created MC blocks as MainchainHeaders and MainchainReferenceData
+        # First 4 SC blocks. Every block contains 49 MainchainHeaders
         for i in range(4):
             check_mcheaders_amount(49, scblock_ids[i], sc_node)
             for mchash in mcblock_hashes[i * 49 : (i + 1) * 49]:
                 check_mcheader_presence(mchash, scblock_ids[i], sc_node)
-            check_mcreferencedata_amount(3, scblock_ids[i], sc_node)
-            for mchash in mcblock_hashes[i * 3 : (i + 1) * 3]:
-                check_mcreferencedata_presence(mchash, scblock_ids[i], sc_node)
+            #check_mcreferencedata_amount(3, scblock_ids[i], sc_node)
```

Comment:
At the moment we can't predict the exact number of RefData to be included, because the RefData collection from the MC Node is limited in time.
Do we want to have any other check instead?